backend/src/test/java/com/daedan/festabook/announcement/service/AnnouncementServiceTest.java

```diff
@@ -0,0 +1,97 @@
+package com.daedan.festabook.announcement.service;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.BDDMockito.given;
+import static org.mockito.Mockito.mock;
+
+import com.daedan.festabook.announcement.domain.Announcement;
+import com.daedan.festabook.announcement.domain.DateTimeGenerator;
+import com.daedan.festabook.announcement.dto.AnnouncementRequest;
+import com.daedan.festabook.announcement.dto.AnnouncementResponse;
+import com.daedan.festabook.announcement.infrastructure.AnnouncementJpaRepository;
+import java.time.LocalDate;
+import java.time.LocalTime;
+import java.util.List;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+
+class AnnouncementServiceTest {
+
+    private final AnnouncementJpaRepository announcementJpaRepository;
+    private final DateTimeGenerator dateTimeGenerator;
+    private final AnnouncementService announcementService;
+
+    public AnnouncementServiceTest() {
+        this.announcementJpaRepository = mock(AnnouncementJpaRepository.class);
+        this.dateTimeGenerator = mock(DateTimeGenerator.class);
+        this.announcementService = new AnnouncementService(announcementJpaRepository, dateTimeGenerator);
+    }
+
```

Comment:
생성자를 확인하기 전 필드 값만 보면 이 필드가 mock 필드인지 확인하기 어려워.

Mockito 어노테이션을 사용하면 해당 필드가 mock을 사용한 값인지 빠르게 인지할 수 있어.
```suggestion
   
@ExtendWith(MockitoExtension.class) // Mockito 어노테이션 활성화
class AnnouncementServiceTest {

    @Mock // 이 필드를 Mock 객체로 만들기
    private AnnouncementJpaRepository announcementJpaRepository;

    @Mock // 이 필드를 Mock 객체로 만들기
    private DateTimeGenerator dateTimeGenerator;

    @InjectMocks // Mock, spy 객체를 주입하여 인스턴스 생성
    private AnnouncementService announcementService;
```
mock, spy, @InjectMocks 자세한 내용은 다음 블로그 참고해 봐도 좋을 듯
https://curiousjinan.tistory.com/entry/spring-unit-test-mocking#3.%20%40InjectMocks%2C%20%40Mock%EC%9D%B4%20%EB%AD%94%EB%8D%B0%3F-1

이건 컨벤션으로 이야기 나눠봐야 할 내용같아.